bindgen-cli/build.rs

```diff
@@ -0,0 +1,49 @@
+use std::char;
+use std::env;
+use std::ffi::OsStr;
+use std::fs::{self, File};
+use std::io::Write;
+use std::path::{Path, PathBuf};
+
+pub fn main() {
+    let out_dir = PathBuf::from(env::var("OUT_DIR").unwrap());
+    let mut dst = File::create(Path::new(&out_dir).join("tests.rs")).unwrap();
+
+    let manifest_dir = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
+    let headers_dir = manifest_dir.join("tests").join("headers");
+
+    let headers = match fs::read_dir(headers_dir) {
+        Ok(dir) => dir,
+        // We may not have headers directory after packaging.
+        Err(..) => return,
+    };
+
+    let entries =
+        headers.map(|result| result.expect("Couldn't read header file"));
+
+    println!("cargo:rerun-if-changed=tests/headers");
+
+    for entry in entries {
```

Comment:
Do you need this? If so, why do we need the `bindgen-tests` crate?

Reply:
the crate exists so `bindgen` does not depend on `bindgen-cli` for the tests.

Reply:
I meant the build script, is this really doing something? This seems to be trying to read `$CARGO_MANIFEST_DIR/tests/headers`, which no longer exists... So can't we just remove it?

Reply:
isn't `$CARGO_MANIFEST_DIR` just `bindgen-tests/`? I printed earch `DirEntry` in the loop and `cargo build -vv -p bindgen-tests` is printing stuff so it seems the path is the right one: `bindgen-tests/tests/headers`

Reply:
Are you looking at `bindgen-cli/build.rs` or `bindgen-tests/build.rs`?

Reply:
(I don't understand why both are needed)

Reply:
ohhhh, that actually explains a lot, my bad :sweat_smile: . Yeah It seems I forgot to remove it